yew-functional-macro/src/lib.rs

```diff
@@ -0,0 +1,144 @@
+use proc_macro::TokenStream as TokenStream1;
```

Comment:
don't do this, please. Just use `proc_macro::TokenStream` in the signature, that's much less confusing.
You can just call `.into()` on `quoted` so you don't have to use it there.

Reply:
Oh, okay. Fixed that.